packages/@okta/vuepress-site/use_cases/inline_hooks/index.md

```diff
@@ -121,7 +122,37 @@ The names of commands follow Java-style reverse DNS name format, beginning with
 
 Lets you return error messages. How the error data is used varies by inline hook type.
 
-Within an `error` object, you need to provide an `errorSummary` property set to a text string. Additionally, you can use an `errorCauses` object to supply more information. A single error object can contain multiple `errorCauses` objects. The fields within errorCauses are: `errorSummary`, `reason`, `locationType`, `location`, and `domain`.
+The `error` object should have the following structure:
+
+| Property     | Description                             | Data Type            |
+|--------------|-----------------------------------------|----------------------|
+| errorSummary | Human-readable summary of the error(s). | String               |
+| errorCauses  | An array of ErrorCause objects.         | Array of ErrorCauses |
+
+The `errorSummary` should be a general statement of any problem the external service encountered in handling the request from Okta. The `errorCauses` are intended to provide more detailed information and are particularly helpful if there were multiple problems.
+
+An `ErrorCause` object must include the following fields:
+
+| Property     | Description                                                                                                                                                         | Data Type |
+|--------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| errorSummary | Human-readable summary of the error.                                                                                                                                | String    |
```

Comment:
Do we want to include somewhere (in this description?) that this field is what will be shown to the end-user? At least for Registration Inline Hooks... not sure if it's relevant for all hooks.